sdk/python/tests/compiler/testdata/watson_train_serve_pipeline.py

```diff
@@ -0,0 +1,103 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# generate default secret name
+import os
+import kfp
+from kfp import components
+from kfp import dsl
+
+secret_name = 'kfp-creds'
+configuration_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/ibm-components/commons/config/component.yaml')
+train_op = components.load_component_from_url('https://raw.githubusercontent.com/tomcli/pipelines/master/components/ibm-components/watson/train/component.yaml')
```

Comment:
`tomcli/pipelines` ... can this point to a public repo? (same below)

Reply:
Thanks, this is pending on another pr that has the necessary changes. I will update this once we get more information tomorrow.

Reply:
ah, right, I should have connected the dots here ...you mentioned it in your opening PR comment  😊